policyengine_us/variables/household/income/person/pensions/military_retirement_pay_survivors.py

```diff
@@ -0,0 +1,9 @@
+from policyengine_us.model_api import *
+
+
+class military_retirement_pay_survivors(Variable):
+    value_type = float
+    entity = Person
+    label = "Military retirement income pay survivors"
```

Comment:
clarify this - is it military retirement pay paid to surviving spouses?

Reply:
Yes, it is.

Reply:
clarify in the label